tests/test_pow2.py

```diff
@@ -25,5 +25,10 @@ def test_interpolated(taso_256_lpf, output_30hz):
 
 
 def test_32hz(input_32hz, output_30hz):
-    resampled = resample_to_30hz(input_32hz, 32)
+    resampled = resample_to_30hz(input_32hz, sample_rate=32)
     assert np.abs(resampled - output_30hz).max() == pytest.approx(0)
+
+
+def test_32hz_interpolate(input_32hz, output_30hz):
+    resampled = resample_to_30hz(input_32hz, sample_rate=32, interpolate=True)
+    assert np.abs(resampled - output_30hz).max() < 1
```

Comment:
We're allowing for max 1-g difference between expected and obtained?
Shall we test for mean, std, etc.?